File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+
+The following code example shows how to modify the footnote/endnote content from the word document.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
```

Comment:
Every XML comment should end with period.

Also, Verb should not be Verb+s.  use, the normal verb

Reply:
I have done so